docs/compilers/osv.md

```diff
@@ -1,10 +1,23 @@
 # OSv Unikernels
 
-UniK uses OSv as a platform for compiling Java to unikernels.
+UniK supports [OSv](http://osv.io) unikernel creation.
+OSv is somewhat different from other unikernel implementations.
+Firstly, it can run application written in any programming language.
+And secondly, due to some specifics in OSv kernel implementation, it can make use of
+precompiled application packages that are stored in public repository.
+In other words, in OSv you can not only run your own application (e.g. your NodeJS server),
+but also arbitrary application from the public repository (e.g. MySQL).
+
+UniK provides support for three languages: nodejs, java and native.
+Native language refers to C/C++ compiled code that can be either your own
+either from the remote repository.
+
+Please note that UniK **does not** compile any of your application. Basically, all that it can
+do for you is to upload your files to the unikernel.
```

Comment:
Please separate here. For Java, it will still compile the code.

Node and dynamic on the other hand, need to contain the files that are to be uploaded into the unikernel.